code/WorkInProgress/MechanicMadness.dm

```diff
@@ -1897,6 +1897,7 @@
 		SEND_SIGNAL(src,COMSIG_MECHCOMP_ADD_INPUT,"add item", "additem")
 		SEND_SIGNAL(src,COMSIG_MECHCOMP_ADD_INPUT,"remove item", "remitem")
 		SEND_SIGNAL(src,COMSIG_MECHCOMP_ADD_INPUT,"remove all items", "remallitem")
+		SEND_SIGNAL(src,COMSIG_MECHCOMP_ADD_INPUT,"remove current + send", "popitem")
```

Comment:
Shouldn't this be named "send current + remove" or "pop current"?

Reply:
yeah, it could be worded more clearly.